packages/block-editor/src/components/block-list/style.scss

```diff
@@ -448,7 +448,7 @@
 	// Left
 	&[data-align="left"] {
 		// This is in the editor only; the image should be floated on the frontend.
-		.block-editor-block-list__block-edit {
+		> .block-editor-block-list__block-edit {
```

Comment:
```suggestion
		.block-editor-block-list__block-edit {
```

Reply:
You can just click the "commit suggestion" here. I can't do it myself as I don't have rights on your repository.